x-pack/plugins/infra/server/lib/alerting/common/messages.ts

```diff
@@ -211,6 +219,6 @@ export const viewInAppUrlActionVariableDescription = i18n.translate(
   'xpack.infra.metrics.alerting.viewInAppUrlActionVariableDescription',
   {
     defaultMessage:
-      'Link to the view or feature within Elastic that can be used to investigate the alert and its context further',
+      'Link to the view or feature within Elastic that can assist with further investigation',
```

Comment:
Changed based on conversation with @vinaychandrasekhar and @maciejforcone on https://github.com/elastic/kibana/issues/139923#issuecomment-1262206576